packages/nextjs-mf/src/plugins/NextFederationPlugin/index.ts

```diff
@@ -9,20 +9,16 @@ import type {
   ModuleFederationPluginOptions,
   NextFederationPluginExtraOptions,
   NextFederationPluginOptions,
-  SharedObject,
 } from '@module-federation/utilities';
-import { Compiler } from 'webpack';
-import { createRuntimeVariables } from '@module-federation/utilities';
+import type { Compiler } from 'webpack';
```

Comment:
Should this use the normalize webpack path here?

Reply:
Type imports are not a issue, since they do not compile down into source code. 
The issue is "real" imports or requires that get to user code. It will require the nearest webpack in node modules, not always the singleton of the compilations version of webpack